vamb/__main__.py

```diff
@@ -2206,6 +2250,20 @@ def add_predictor_arguments(subparser: argparse.ArgumentParser):
     return subparser
 
 
+def add_cluster_only_args(subparser: argparse.ArgumentParser):
+    c_only_arg = subparser.add_argument_group(
+        title="Clustering options", description=None
+    )
+    c_only_arg.add_argument(
+        "--latent",
+        dest="latent_file",
+        required=True,
+        metavar="",
+        type=lambda p: np.load(Path(p)),
```

Comment:
Set the type to Path, and then write your own logic to load the file using `vamb.vambtools.read_npz` elsewhere, below the line `elif args.partial_part == "cluster":`.
This enables better error messages (i.e. it makes it easier to distinguish between passing a path that doesn't exist, and a path that doesn't point to an `.npz` file)